trie/statistics/trieStatistics.go

```diff
@@ -146,18 +145,8 @@ type TrieStatsDTO struct {
 
 // ToString returns the collected statistics as a string array
 func (tsd *TrieStatsDTO) ToString() []string {
-	addressConverter, err := pubkeyConverter.NewBech32PubkeyConverter(addressLength, log)
-	if err != nil {
-		log.Error("could not create address converter", "err", err.Error())
-	}
-
-	address := ""
-	if len(tsd.Address) == addressLength {
-		address = addressConverter.Encode(tsd.Address)
-	}
-
 	stats := make([]string, 0)
-	stats = append(stats, fmt.Sprintf("address %v,", address))
+	stats = append(stats, fmt.Sprintf("address %v,", tsd.Address))
```

Comment:
do we have tsd.Address as string or bytes? Because if we have as bytes, %v will print something different than a bech32-string

Reply:
Good point. It was stored as `[]byte`, will change to `string`